client/store/admin.js

```diff
@@ -0,0 +1,85 @@
+import axios from 'axios'
+// import history from '../history'
+
+//INITIAL STATE
+const initialState = {
+  allUsers: [],
+  selectedUser: {}
```

Comment:
selectedUser field in reducer is set to string - consider changing selectedUser in initialState to empty string